packages/node/test/unit/channels.spec.ts

```diff
@@ -0,0 +1,48 @@
+import * as cf from "@counterfactual/cf.js";
+import { ethers } from "ethers";
+import * as _ from "lodash";
+
+import { Node, StateChannelInfoImpl } from "../../src/node";
+
+describe("Node can contain channels", () => {
+  it("Node has no channels", () => {
```

Comment:
Should write unit tests for basic instantiation e.g., https://github.com/gnosis/pm-js/blob/master/test/test_gnosis.js#L17-L35